lib/src/server/handler.dart

```diff
@@ -293,7 +293,13 @@ class ServerHandler_ extends ServiceCall {
     _customTrailers = null;
     outgoingTrailersMap['grpc-status'] = status.toString();
     if (message != null) {
-      outgoingTrailersMap['grpc-message'] = message;
+      try {
+        outgoingTrailersMap['grpc-message'] =
+            Uri.encodeFull(message).replaceAll("%20", " ");
+      } catch (_) {
```

Comment:
It is not clear to me what exceptions we can get here.

https://api.dart.dev/stable/2.8.4/dart-core/Uri/encodeFull.html doesn't mention any exceptions

Reply:
You're right. Encoding string does not throw any exception. I fixed it.